docs/content/direct/core-chart-step-by-step-installation.md

```diff
@@ -0,0 +1,340 @@
+```
+
+## Step 2: Check Container Name
+
+verify the container name that will be needed for the Helm chart configuration:
+
+```bash
+docker ps --filter "name=kubeflex" --format "{{.Names}}"
+```
+
+**Expected output (similar to):**
+```
+kubeflex-control-plane
+```
+
+## Step 3: Update Chart Dependencies
+
+navigate to your KubeStellar directory and update the chart dependencies:
+
+```bash
+helm dependency update core-chart
+```
+
+**Expected output (similar to):**
+```
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "antrea" chart repository
+...Successfully got an update from the "prometheus-community" chart repository
+...Successfully got an update from the "istio" chart repository
+Update Complete. ⎈Happy Helming!⎈
+Saving 2 charts
+Downloading kubeflex-operator from repo oci://ghcr.io/kubestellar/kubeflex/chart
+Pulled: ghcr.io/kubestellar/kubeflex/chart/kubeflex-operator:v0.8.9
+Digest: sha256:2be43de71425ad682edca6544f6c3a5864afbfad09a4b7e1e57bde6dae664334
+Downloading argo-cd from repo oci://ghcr.io/argoproj/argo-helm
+Pulled: ghcr.io/argoproj/argo-helm/argo-cd:7.8.5
+Digest: sha256:662f4687e8e525f86ff9305020632b337a09ffacb7b61b7c42a841922c91da7b
+Deleting outdated charts
+```
+
+## Step 4: Initial Chart Installation
+
+first, install the basic chart to set up KubeFlex:
+
+```bash
+helm upgrade --install ks-core core-chart
+```
+
+**Expected output (similar to):**
+```
+Release "ks-core" does not exist. Installing it now.
+NAME: ks-core
+LAST DEPLOYED: Tue Jun  3 09:15:08 2025
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+For your convenience you will probably want to add contexts to your
+kubeconfig named after the non-host-type control planes (WDSes and
+ITSes) that you just created (a host-type control plane is just an
+alias for the KubeFlex hosting cluster). You can do that with the
+following `kflex` commands; each creates a context and makes it the
+current one. See
+https://github.com/kubestellar/kubestellar/blob/0.28.0-alpha.2/docs/content/direct/core-chart.md#kubeconfig-files-and-contexts-for-control-planes
+for a way to do this without using `kflex`.
+Start by setting your current kubeconfig context to the one you used
+when installing this chart.
+
+kubectl config use-context $the_one_where_you_installed_this_chart
+kflex ctx --set-current-for-hosting # make sure the KubeFlex CLI's hidden state is right for what the Helm chart just did
+
+
+Finally, you can use `kflex ctx` to switch back to the kubeconfig
+context for your KubeFlex hosting cluster.
+```
+
+## Step 5: Verify Current Context
+
+check your current kubectl context:
+
+```bash
+kubectl config get-contexts
+```
+
+**Expected output (similar to):**
+```
+CURRENT   NAME            CLUSTER         AUTHINFO        NAMESPACE
+*         kind-kubeflex   kind-kubeflex   kind-kubeflex   
+```
+
+## Step 6: Install KubeStellar Core with Control Planes
```

Comment:
Comparing the content here with https://docs.kubestellar.io/release-0.28.0-alpha.2/direct/get-started/#use-core-helm-chart-to-initialize-kubeflex-and-create-its-and-wds , the first biggest difference is the use of the local copy of the chart vs. the use of a released chart image.

We have deliberately not required users to `git clone` the kubestellar repo. You can add that as an option, but I would not want to remove the option to use released stuff.